lib/SimpleSAML/Logger.php

```diff
@@ -288,6 +288,15 @@ public static function getCapturedLog(): array
     }
 
 
+    /**
+     * Clears the captured log.
+     */
+    public static function clearCapturedLog()
+    {
+        return self::$capturedLog = [];
```

Comment:
Unused return value, should probably not 'return' the empty array but just void, and also specify that as the return type